README.md

```diff
@@ -640,6 +639,7 @@ Yazılım geliştirme ile ilgili özenle seçilmiş Türkçe kaynaklar listesi.
 - [Yazılım Üzerine Yazılmış En İyi 5 Kitap](https://www.onbirkod.com/yazilim-uzerine-yazilmis-en-iyi-5-kitap/) - [Talat Tüfekçi](https://www.onbirkod.com)
 - [Apache JMeter: Web Uygulamamızın Yük Testlerini Nasıl Yaparız? (1): Giriş, Kurulum](https://www.onbirkod.com/apache-jmeter-web-uygulamamizin-yuk-testlerini-nasil-yapariz-1/) - [Talat Tüfekçi](https://www.onbirkod.com)
 - [Apache JMeter: Web Uygulamamızın Yük Testlerini Nasıl Yaparız? (2): Basit Testler](https://www.onbirkod.com/apache-jmeter-web-uygulamamizin-yuk-testlerini-nasil-yapariz-2/) - [Talat Tüfekçi](https://www.onbirkod.com)
+- [hacker-laws (Programcıların faydalı bulacağı yasalar, teoriler, prensipler ve desenler.)](https://github.com/dwmkerr/hacker-laws) - [Umut Işık](https://umuts.info)
```

Comment:
http://umuts.info/hacker-laws-tr/

Reply:
Orda farklı bir yapıya gidiyor repo sahibi. Onun için direk repo linki verdim.

Reply:
Türkçe Readme dosyasının linkini verebiliriz istersen.

https://github.com/dwmkerr/hacker-laws/blob/master/translations/tr.md

Reply:
@gokaygurcan 
Söylediğin şekilde düzeltim. Ana repo sahibi yapıyı değiştirdiğinde tekrar güncellerim.

Reply:
Merhaba,
Amaç doğru içeriğe kolayca erişmek olduğu için link'lerin düzeltilmesini teklif etmiştim. Yoksa açıp translations klasörünün altından tr.md'yi bulmak zor olduğu için değil. Önemli nokta şu: bu içeriği belki hayatında hiç GitHub kullanmamış, belki ilk kez PHP ile uğraşmaya başlamış ve Google aramasından gelmiş birisi de okuyor olabilir. 

Rule of thumb şu: basit ve anlaşılır, hap bilgi dediğimiz türden içerik + en doğru, en kolay ve en güvenilir ulaşım yöntemi. bu yüzden her link'i incelemeye, yorum yapmaya çalışıyoruz ve bu yüzden her yeni PR ile tüm link'lerin ayakta ve ulaşılabilir olduğundan emin olmaya çalışıyoruz, olmayanları listeden kaldırıyoruz.